src/Client/LightningStore.php

```diff
@@ -152,21 +153,33 @@ public function getInvoice(
         }
     }
 
+    /**
+     * Amount wrapped in a string, represented in a millistatoshi string.
+     * (1000 millisatoshi = 1 satoshi.
+     *
+     * @param string $amount
+     */
     public function payLightningInvoice(
         string $cryptoCode,
         string $storeId,
-        string $BOLT11
+        string $BOLT11,
+        ?string $amount = null,
+        ?string $maxFeePercent = null,
```

Comment:
To avoid PHP rounding errors with float we introduced PreciseNumber class which we could use here too. On the other hand it is very unlikely that somebody will submit a percentage with more than 2 decimals, so there should be no rounding issue anyway and we could have kept it float. String is also ok I guess, not sure what the api does when you pass '2' instead of '2.0' afair it will do just fine.